sr_demos/scripts/sr_demos/demo.py

```diff
@@ -393,7 +400,7 @@ def sequence_lf(hand_commander, joint_states_config, tactile_reading):
     rospy.sleep(0.5)
     # Initialize wake time
     wake_time = time.time()
-    CONST_TIME_TO_COMPLETE_DEMO = 15
+    const_time_to_complete_demo = 15
```

Comment:
This doesn't seem to be ever modified, so why reformating from upper case to lower case?

Reply:
Constants cannot be initalised in functions according to PEP8